### ISSUE TITLE
Usability improvments

### ISSUE THREAD

**Issue by tukataka** [okta/okta-sdk-php#74]
I want to use method `getUsers` for getting all users from my account. My account have more, than 200 users. I need to get link to next page in headers. But right now header doesn't save to some place (as is response). 
As I understand only way to do it is set custom httpClient. It's very uncomfortable. Can you improve this moment ?

**Comment by bretterer** [comment]
@tukataka - Yes, that is unfortunately a limitation of this particular SDK. The best I can tell you for now is to use your own custom httpClient.  We will be updating the SDK and will make sure to address this at that time. 

I do not have a definitive time this will be finished, so please keep an eye on this ticket for updates.

**Comment by bretterer** [comment]
Internal Ref: OKTA-291534

**Comment by MrJamesEllis** [comment]
Agree, the only way to inspect headers is to provide a custom HttpClient to the ClientBuilder that can override sendRequest and store the response.
